pytest_tests/helpers/s3_helper.py

```diff
@@ -127,3 +130,27 @@ def assert_object_lock_mode(
         assert (
             retain_date - last_modify + timedelta(seconds=1)
         ).days == retain_period, f"Expected retention period is {retain_period} days"
+
+
+def assert_s3_acl(acl_grants: dict, permitted_users: str):
+    if permitted_users == "AllUsers":
+        for i in range(len(acl_grants)):
```

Comment:
Why to add complexity and iterate using `i`?
Can we use this instead?
```
for acl_grant in acl_grants:
     acl_grant.get(...) ... 
    ...
```

Reply:
Soon, I'll commit a change to this block.
But the logic here is that not every `acl_grant` contains permission for All Users, others may contain "Canonical User".

Reply:
My comment only relates to style of iteration.
Basically, instead of 
```
for i in range(len(Xs)):
     x[i] <...>
```

It is better to use
```
for x in Xs:
     x <...>
```

Because we are working only with one instance of Xs at a time. 
We don't do something like
```
Xs[i+1] = Xs[i] 
if Xs[i-1] > Xs[i]
```
And we don't need `i` elsewhere either.

Reply:
Done